README.md
# dotfiles
Linux config and dotfiles


ranger/history
rename 
rename Fight Club [1999] - 10th Anniversary Edition.srt
delete 
h
shell mkdir Rocky 2 [197x]
rename Rocky 2 [1979]
rename Rocky 2 [1979].mkv
rename Rocky 2 [1979].srt
rename Rocky II [1979]
rename Rocky II [1979].mp4
rename Rocky II [1979].srt
shell 
open_with 
shell mkdir 4
shell touch prosti.cpp
shell touch faktor.cpp
shell touch kabonacci.cpp
rename prosti2.cpp
shell mkdir ispit
shell mkdir mi2014\/2015
shell mkdir mi14-15
shell cd
shell sudo chmod 666 Elektronika\ 1\ -\ Butkovic,\ Divkovic,\ Baric\ -\ Skripta\ -\ 2.\ ciklus.pdf
shell chmod 664 Elektronika\ 1\ -\ Butkovic,\ Divkovic,\ Baric\ -\ Skripta\ -\ 2.\ ciklus.pdf
cd 
hghjk
shell touch 2015-12-11-01.txt
shell touch 2015-12-15-01.txt
rename .diary
rename RETURN_OF_THE_JEDI_1983.en.sub
rename RETURN_OF_THE_JEDI_1983.sub.en
rename RETURN_OF_THE_JEDI_1983.sub.wtf
rename RETURN_OF_THE_JEDI_1983.sub
rename RETURN_OF_THE_JEDI_1983.en.srt
rename RETURN_OF_THE_JEDI_1983.srt
shell touch 2015-12-16-01.txt
shell mkdir zadaci
shell touch foton.cpp
touch foton.sage
open_with firefox
open_with vim
shell touch 2015-12-18-01.txt
touch 2015-12-25-01.txt
touch 2015-12-26-01.txt
shell mkdir 2016-01
shell touch 2016-01-01-01.txt
delete
rename Rocky Balboa [2006]

rename Limitless [2011]


ranger/colorschemes/test.py
# This file is part of ranger, the console file manager.
# License: GNU GPL version 3, see the file "AUTHORS" for details.
# Author: Joseph Tannhuber <sepp.tannhuber@yahoo.de>, 2013
# Solarized like colorscheme, similar to solarized-dircolors
# from https://github.com/seebi/dircolors-solarized.
# This is a modification of Roman Zimbelmann's default colorscheme.

from ranger.gui.colorscheme import ColorScheme
from ranger.gui.color import *

class Solarized(ColorScheme):
    progress_bar_color = 33

    def use(self, context):
        fg, bg, attr = default_colors

        if context.reset:
            return default_colors
	
	fg = 174
	bg = 0

#        elif context.in_browser:
#            fg = 4
#            if context.selected:
#                attr = reverse
#            else:
#                attr = normal
#            if context.empty or context.error:
#                fg = 235
#                bg = 160
#            if context.border:
#                fg = default
#            if context.media:
#                if context.image:
#                    fg = 136
#                else:
#                    fg = 166
#            if context.container:
#                fg = 61
#            if context.directory:
#                fg = 33
#            elif context.executable and not \
#                    any((context.media, context.container,
#                        context.fifo, context.socket)):
#                fg = 64
#                attr |= bold
#            if context.socket:
#                fg = 136
#                bg = 230
#                attr |= bold
#            if context.fifo:
#                fg = 136
#                bg = 230
#                attr |= bold
#            if context.device:
#                fg = 244
#                bg = 230
#                attr |= bold
#            if context.link:
#                fg = context.good and 37 or 160
#                attr |= bold
#                if context.bad:
#                    bg = 235
#            if context.tag_marker and not context.selected:
#                attr |= bold
#                if fg in (red, magenta):
#                    fg = white
#                else:
#                    fg = red
#            if not context.selected and (context.cut or context.copied):
#                fg = 234
#                attr |= bold
#            if context.main_column:
#                if context.selected:
#                    attr |= bold
#                if context.marked:
#                    attr |= bold
#                    fg = 144
#                    bg = 0
#            if context.badinfo:
#                if attr & reverse:
#                    bg = magenta
#                else:
#                    fg = magenta
#
#        elif context.in_titlebar:
#            attr |= bold
#            if context.hostname:
#                fg = context.bad and 45  or 255
#                if context.bad:
#                    bg = 45
#            elif context.directory:
#                fg = 255
#            elif context.tab:
#                fg = context.good and 45 or 33
#                bg = 39
#            elif context.link:
#                fg = yellow
#
#        elif context.in_statusbar:
#            if context.permissions:
#                if context.good:
#                    fg = 93
#                elif context.bad:
#                    fg = 160
#                    bg = 235
#            if context.marked:
#                attr |= bold | reverse
#                fg = 237
#                bg = 47
#            if context.message:
#                if context.bad:
#                    attr |= bold
#                    fg = 160
#                    bg = 235
#            if context.loaded:
#                bg = self.progress_bar_color

#        if context.text:
#            if context.highlight:
#                attr |= reverse

#        if context.in_taskview:
#            if context.title:
#                fg = 93

#            if context.selected:
#                attr |= reverse

#            if context.loaded:
#                if context.selected:
#                    fg = self.progress_bar_color
#                else:
#                    bg = self.progress_bar_color

        return fg, bg, attr


ranger/colorschemes/redgold.py
# This file is part of ranger, the console file manager.
# License: GNU GPL version 3, see the file "AUTHORS" for details.
# Author: Joseph Tannhuber <sepp.tannhuber@yahoo.de>, 2013
# Solarized like colorscheme, similar to solarized-dircolors
# from https://github.com/seebi/dircolors-solarized.
# This is a modification of Roman Zimbelmann's default colorscheme.

from ranger.gui.colorscheme import ColorScheme
from ranger.gui.color import *

class Solarized(ColorScheme):
    progress_bar_color = 33

    def use(self, context):
        fg, bg, attr = default_colors

        if context.reset:
            return default_colors

#        elif context.in_browser:
#            fg = 4
#            if context.selected:
#                attr = reverse
#            else:
#                attr = normal
#            if context.empty or context.error:
#                fg = 235
#                bg = 160
#            if context.border:
#                fg = default
#            if context.media:
#                if context.image:
#                    fg = 136
#                else:
#                    fg = 166
#            if context.container:
#                fg = 61
#            if context.directory:
#                fg = 33
#            elif context.executable and not \
#                    any((context.media, context.container,
#                        context.fifo, context.socket)):
#                fg = 64
#                attr |= bold
#            if context.socket:
#                fg = 136
#                bg = 230
#                attr |= bold
#            if context.fifo:
#                fg = 136
#                bg = 230
#                attr |= bold
#            if context.device:
#                fg = 244
#                bg = 230
#                attr |= bold
#            if context.link:
#                fg = context.good and 37 or 160
#                attr |= bold
#                if context.bad:
#                    bg = 235
#            if context.tag_marker and not context.selected:
#                attr |= bold
#                if fg in (red, magenta):
#                    fg = white
#                else:
#                    fg = red
#            if not context.selected and (context.cut or context.copied):
#                fg = 234
#                attr |= bold
#            if context.main_column:
#                if context.selected:
#                    attr |= bold
#                if context.marked:
#                    attr |= bold
#                    fg = 144
#                    bg = 0
#            if context.badinfo:
#                if attr & reverse:
#                    bg = magenta
#                else:
#                    fg = magenta
#
#        elif context.in_titlebar:
#            attr |= bold
#            if context.hostname:
#                fg = context.bad and 45  or 255
#                if context.bad:
#                    bg = 45
#            elif context.directory:
#                fg = 255
#            elif context.tab:
#                fg = context.good and 45 or 33
#                bg = 39
#            elif context.link:
#                fg = yellow
#
#        elif context.in_statusbar:
#            if context.permissions:
#                if context.good:
#                    fg = 93
#                elif context.bad:
#                    fg = 160
#                    bg = 235
#            if context.marked:
#                attr |= bold | reverse
#                fg = 237
#                bg = 47
#            if context.message:
#                if context.bad:
#                    attr |= bold
#                    fg = 160
#                    bg = 235
#            if context.loaded:
#                bg = self.progress_bar_color

#        if context.text:
#            if context.highlight:
#                attr |= reverse

#        if context.in_taskview:
#            if context.title:
#                fg = 93

#            if context.selected:
#                attr |= reverse

#            if context.loaded:
#                if context.selected:
#                    fg = self.progress_bar_color
#                else:
#                    bg = self.progress_bar_color

        return fg, bg, attr
